cdap-ui/app/cdap/components/DataPrep/Directives/DropColumn/index.js

```diff
@@ -26,7 +27,7 @@ export default class DropColumnDirective extends Component {
   }
 
   applyDirective() {
-    let column = this.props.column;
+    let column = this.props.column.toString();
```

Comment:
Array.toString() would return comma separated string already I believe, so should be OK